.isort.cfg

```diff
@@ -10,3 +10,4 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
+ensure_newline_before_comments = True
```

Comment:
Possibly we should just dump most options here and add:

```
profile = "black"
```

See https://pycqa.github.io/isort/docs/configuration/black_compatibility/

(Note: in v13 branches, we used older isort, which didn't have that option).